src/cert_store.rs

```diff
@@ -150,6 +151,34 @@ impl CertStore {
         }
     }
 
+    /// Deletes a specified store within the context of the current user.
+    /// 
+    /// Deleting stores is not natural and should only be done if necessary
```

Comment:
Not sure what 'natural' means here, could you explain why it shouldn't usually be done?